test/e2e

```diff
@@ -129,9 +139,25 @@ test_rhel_version() {
 list_kernel_packages() {
     # Check that DTK contains all the packages
     oc debug --image-stream="openshift/driver-toolkit:latest" \
-	    -n openshift \
-            --quiet \
-            -- dnf list installed | grep kernel
+	     -n openshift \
+             --quiet \
+             -- dnf list installed \
+        | grep kernel \
+        | tee ${ARTIFACT_DIR}/dnf-list-installed-kernel
+}
+
+get_dtk_image_info() {
```

Comment:
@kpouget this function is currently unused?

Reply:
good catch, fixed

I also removed the `"` from this file: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_driver-toolkit/69/pull-ci-openshift-driver-toolkit-master-e2e-aws-driver-toolkit/1465712977075245056/artifacts/e2e-aws-driver-toolkit/test/artifacts/latest_image.address

Reply:
hum, not sure if we keep this function, the output doesn't look as exected:

see [env](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_driver-toolkit/69/pull-ci-openshift-driver-toolkit-master-e2e-aws-driver-toolkit/1465726375619989504/artifacts/e2e-aws-driver-toolkit/test/artifacts/dtk_image_env) and [git version](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_driver-toolkit/69/pull-ci-openshift-driver-toolkit-master-e2e-aws-driver-toolkit/1465726375619989504/artifacts/e2e-aws-driver-toolkit/test/artifacts/git_commit) from the test, vs what I have in my cluster image: 
```
oc debug --image-stream=openshift/driver-toolkit:latest -n openshift --quiet
sh-4.4# env
OS_GIT_MINOR=9
SOURCE_GIT_URL=https://github.com/openshift/driver-toolkit
HOSTNAME=image-debug
OS_GIT_TREE_STATE=clean
BUILD_VERSION=v4.9.0
OS_GIT_COMMIT=25c3513
KUBERNETES_PORT_443_TCP_PROTO=tcp
KUBERNETES_PORT_443_TCP_ADDR=172.30.0.1
container=oci
KUBERNETES_PORT=tcp://172.30.0.1:443
PWD=/tmp/kmods-via-containers
HOME=/root
OS_GIT_VERSION=4.9.0-202110182323.p0.git.25c3513.assembly.stream-25c3513
OS_GIT_MAJOR=4
KUBERNETES_SERVICE_PORT_HTTPS=443
KUBERNETES_PORT_443_TCP_PORT=443
SOURCE_GIT_COMMIT=25c351345f6e06130dfd97d79fc417f9bbc7ebbe
KUBERNETES_PORT_443_TCP=tcp://172.30.0.1:443
TERM=xterm
GODEBUG=x509ignoreCN=0,madvdontneed=1
NSS_SDB_USE_CACHE=no
SHLVL=1
KUBERNETES_SERVICE_PORT=443
SOURCE_DATE_EPOCH=1633474441
OS_GIT_PATCH=0
SOURCE_GIT_TREE_STATE=clean
SOURCE_GIT_TAG=25c3513
__doozer=merge
BUILD_RELEASE=202110182323.p0.git.25c3513.assembly.stream
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
KUBERNETES_SERVICE_HOST=172.30.0.1
```

Reply:
As discussed on slack, this is because the presubmit tests are [testing the DTK image](https://github.com/openshift/release/blob/7b45da8dafb8f404c755de33042ffa3e597f0690/ci-operator/config/openshift/driver-toolkit/openshift-driver-toolkit-master.yaml#L25) built in CI, so this is working as expected.